parsers/mysql/mysql.go

```diff
@@ -99,9 +99,11 @@ const (
 
 var (
 	reTime             = parsers.ExtRegexp{regexp.MustCompile("^# Time: (?P<time>[^ ]+)Z *$")}
+	reVCTime           = parsers.ExtRegexp{regexp.MustCompile("^# Time: (?P<datetime>[0-9]+ [0-9:]+)")}
```

Comment:
nit: maybe add a comment explaining why we have these alternate regexes? (VC=vividcortex?)

Reply:
::sigh:: more accurate would be pre-mysql-5.somethingerother... I'll change the name to reflect that.